.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -0,0 +1,109 @@
+name: 🐞 Bug Report
+description: Report a bug
+labels: ['bug']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hi there! :wave: :smiley:
+        Thanks for taking the time to fill out this bug report!
+
+        > To expedite issue processing please search open and closed issues before submitting a new one.
+  - type: dropdown
+    id: package-concerned
+    attributes:
+      label: I am opening an issue for
+      options:
+        - Styleguide documentation
+        - Figma
+        - Icons
+        - Styles (CSS, SCSS, ...)
+        - Tokens
+        - Tools (PostCSS, Stylelint, ...)
+    validations:
+      required: true
+  - type: input
+    id: package-version
+    attributes:
+      label: Package version
+      description: Which version of the package is affected?
+      placeholder: ex. @mozaic-ds/styles@1.41.0
+    validations:
+      required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      placeholder: A clear and concise description of what the bug is
+    validations:
+      required: true
+  - type: input
+    id: github-repository
+    attributes:
+      label: Github repository
+      description: To better help you, please indicate the url of your GitHub repo
+      placeholder: ex. https://github.com/adeo/mozaic-design-system
+  - type: input
+    id: mockups
+    attributes:
+      label: Mockups
+      description: Please indicate the url to your Figma mockups to give us more context
+      placeholder: ex. https://www.figma.com/file/b09kkFmjlH6RnLEDF2vppS/%5BBA%5D-Button-%F0%9F%9F%A2?node-id=0%3A1
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Tell us, what's happening?
+      placeholder: |
+        Tell us what you see
+
+        Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+  - type: textarea
+    id: what-expected
+    attributes:
+      label: What is expected?
+      description: Tell us, what did you expect to happen?
+      placeholder: Describe the expected outcome
```

Comment:
~~Describe the expected outcome~~
Describe the expected outcome.

Reply:
Fixed